src/util.js

```diff
@@ -313,45 +328,113 @@ function isPrimitive (data) {
   )
 }
 
+/**
+ * Searches a key or value in the object, with a maximum deepness
+ * @param {*} obj Search target
+ * @param {string} searchTerm Search string
+ * @returns {boolean} Search match
+ */
 export function searchDeepInObject (obj, searchTerm) {
-  var match = false
+  return internalSearchObject(obj, searchTerm.toLowerCase(), new Map(), 0)
```

Comment:
I think it would be safer to clear the map when leaving the function to avoid holding references to objects and creating a leak